templates/climate_fever/templates.yaml

```diff
@@ -0,0 +1,87 @@
+dataset: climate_fever
+templates:
+  82c484bd-2ed7-4ee0-aaee-2b31ac68e751: !Template
+    id: 82c484bd-2ed7-4ee0-aaee-2b31ac68e751
+    jinja: 'Considering the following claim:
+
+      {{claim}}.
+
+      Does the following statement {{"supports"}}, {{"refutes"}}, or provide {{"not
+      enough info"}} on climate change?
+
+      {{evidences[4]["evidence"].strip(".").strip(''"'')}}.
+
+      |||
+
+      {{ ["SUPPORTS", "REFUTES", "NOT_ENOUGH_INFO"][evidences[4]["evidence_label"]]
```

Comment:
Could you please change it to ["Supports", "Refutes", "Not enough information"]?
same for the other templates

Reply:
Sure will do